chart/hooks/pre-upgrade/remove-deployment-updater-readiness.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -ex
+
+patch="
+---
+spec:
+  template:
+    spec:
+      containers:
+      - name: cc-deployment-updater-cc-deployment-updater
+        readinessProbe:
+          $patch: delete
```

Comment:
I can't seem to figure out where `$patch` is defined (since this is a double-quoted string, it should expand somewhere…)